app/main.py

```diff
@@ -1 +1,25 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
```

Comment:
Code Style: It is recommended to use type annotations in your code. It helps to understand what type of arguments are expected and what type of result is returned. For example, it is not clear what type of object `cafe` is. It's better to specify it explicitly like `cafe: Cafe`.